dist/src/main/config/broker.yaml.template

```diff
@@ -0,0 +1,287 @@
+# Zeebe Broker configuration file (without embedded gateway)
```

Comment:
what is the difference to broker.standalone?

Reply:
Ok the only difference is embedded gateway? I'm asking myself whether this is really necessary to have two separate files for that.

Reply:
That's what we decided to do. The one file is for people who just want to test Zeebe standalone, the other is for people who want to configure nodes in a cluster. The documentation also hints into that direction